crates/astria-conductor/src/sequencer/mod.rs

```diff
@@ -129,7 +128,7 @@ impl Reader {
                             }
 
                             executor::channel::TrySendError::NoPermits(block) => {
-                                trace!("executor channel is full; scheduling block and stopping block fetch until a slot opens up");
+                                debug!("executor channel is full; scheduling block and stopping block fetch until a slot opens up");
```

Comment:
This was intentionally kept at trace-level because it's pretty low level because right now this can trigger extremely frequently == create lots of noise.

Reply:
removed